pkg/utils/parse_arf_result.go

```diff
@@ -450,13 +488,79 @@ func remediationsFromString(scheme *runtime.Scheme, name string, namespace strin
 		})
 	}
 
-	return rems
+	return rems, nil
+}
+
+//This function will take orginal remediation content, and a list of all values found in the configMap
+//It will processed and substitue the value in remediation content, and return processed Remediation content
+//The return will be Processed-Remdiation Content, Value-Used List, Un-Set List, and err if possible
+func parseValues(remContent string, resultValues map[string]string) (string, []string, []string, error) {
+	t, err := template.New("").Option("missingkey=zero").Parse(remContent)
+	var valuesUsedList []string
+	var valuesMissingList []string
+	var valuesParsedList []string
+	if err != nil {
+		return remContent, valuesUsedList, valuesMissingList, errors.New("wrongly formatted remediation context: " + err.Error()) //Error creating template // Wrongly formatted remediation context
+	}
+	buf := &bytes.Buffer{}
+	err = t.Execute(buf, resultValues)
+	fixedText := buf.String()
```

Comment:
fixedText appears to be used only in the error handling, maybe move it there?

Reply:
Sounds good, I will do that.

Reply:
Actually I think `fixedText` is used for other returns as well. the `fixedText` here in the error handling block is just a placeholder. The `fixedText` is parsed remediation context with value instead of variable name from datastream.

Reply:
sorry, I missed it, you're right